### PR TITLE
Child Theme Accommodations & other stuff

### PR DESCRIPTION
**Pull request by kangabell**

### CODE REVIEW

File: library/scripts/equalize-and-expand.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  $(window).resize(function(){
+
+    if (Modernizr.mq('(min-width: 1024px)')) {
+
+			// reset in case it's already been equalized
+			unEqualize('body.home section.hero > div');
+			unEqualize('body.home section.articles article a');
+			unEqualize('body.home section.block');
+
```
Comment (jhogue) [inline library/scripts/equalize-and-expand.js:13]: Small nitpick, looks like the tabs/spaces are misaligned. Not a deal breaker.

**Comment by jhogue** [comment]
I think I get it. With the theme split for multi site, we need to use a different function to define the root, and for easier portability, all the JS has been pulled out into components so they are easier to reuse for another child theme. Looks good to me.

File: library/scripts/equalize-and-expand.js (hunk 1 above)
Comment (kangabell) [reply library/scripts/equalize-and-expand.js:13]: You're right. It was like that originally too – I just copied and pasted stuff, didn't edit at all. Probably won't fix at this time.

**Comment by kangabell** [comment]
Yep pretty much! Merging...

**Pull request merged**